### PR TITLE
Adds a layer for rendering PointCloud2 data in 2D.

### PR DESCRIPTION
**Pull request by damonkohler**

### CODE REVIEW

**Comment by damonkohler** [comment]
@SirVer PTAL

File: android_15/src/org/ros/android/view/visualization/layer/PointCloud2DLayer.java [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+
+    final ChannelBuffer buffer = pointCloud.getData();
+    while (buffer.readable()) {
+      vertexBackBuffer.put(buffer.readFloat());
+      vertexBackBuffer.put(buffer.readFloat());
+      // Discard z data.
+      buffer.readFloat();
+      // Discard intensity.
+      buffer.readFloat();
+      vertexBackBuffer.put(0.f);
```
Comment (SirVer) [inline android_15/src/org/ros/android/view/visualization/layer/PointCloud2DLayer.java]: move this up to the z data.

**Comment by SirVer** [comment]
one nit.

**Comment by damonkohler** [comment]
Fixed.

**Pull request merged**